### PR TITLE
Added InceptionV3

### LINKED ISSUE #571: Include InceptionV3 model
**Short Description**

Adding InceptionV3 model.

**Papers**
https://arxiv.org/abs/1512.00567

**Existing Implementations**

https://github.com/keras-team/keras/blob/master/keras/applications/inception_v3.py

### PR DESCRIPTION
**Pull request by ayyucedemirbas**
# What does this PR do?
Added InceptionV3

Fixes #571  

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
       https://github.com/keras-team/keras-cv/issues/571
- [ ] Did you write any new necessary tests?

## Who can review?

 @LukeWood @tanzhenyu @sayakpaul

### CODE REVIEW

File: keras_cv/models/inception_v3.py [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+from tensorflow.keras import backend
+from tensorflow.keras.applications import imagenet_utils
+from tensorflow.keras.engine import training
+from tensorflow.keras.layers import VersionAwareLayers
+from tensorflow.keras.utils import data_utils
+from tensorflow.keras.utils import layer_utils
+
+# isort: off
+from tensorflow.python.util.tf_export import keras_export
```
Comment (LukeWood) [inline keras_cv/models/inception_v3.py:34]: remove keras_export

File: keras_cv/models/inception_v3.py [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+layers = VersionAwareLayers()
```
Comment (LukeWood) [inline keras_cv/models/inception_v3.py:36]: don't need version aware layers in KerasCV

File: keras_cv/models/inception_v3.py [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+def InceptionV3(
```
Comment (LukeWood) [inline keras_cv/models/inception_v3.py:38]: please update the API to match the example set in DenseNet.  there are distinctions between keras.applications and keras_cv.models:

https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/densenet.py

**Review by LukeWood — CHANGES_REQUESTED** [review]
Thanks for the PR!

please update the API and code style to match keras_cv.models (no input_tensor, include_rescaling as an argument, include_top as required, no channels_first support, etc.)

After doing this we can do a full review.

**Comment by ayyucedemirbas** [comment]
Thanks, @LukeWood ! All solved!

File: keras_cv/models/inception_v3.py [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+
+from tensorflow.keras import backend
+from tensorflow.keras.applications import imagenet_utils
+from tensorflow.keras.engine import training
+from tensorflow.keras.utils import data_utils
+from tensorflow.keras.utils import layer_utils
+from tensorflow.keras import layers
+
+
+def InceptionV3(
```
Comment (tanzhenyu) [inline keras_cv/models/inception_v3.py:34]: can you add unit test for the model?

File: keras_cv/models/inception_v3.py [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+            x = layers.GlobalAveragePooling2D()(x)
+        elif pooling == "max":
+            x = layers.GlobalMaxPooling2D()(x)
+
+
+    inputs = img_input
+    # Create model.
+    model = training.Model(inputs, x, name="inception_v3")
+
+    if weights is not None:
```
Comment (tanzhenyu) [inline keras_cv/models/inception_v3.py:359]: can you try this model with the basic_training script we have today, and report the result?

**Review by tanzhenyu — COMMENTED** [review]
Thanks for the PR!

**Pull request closed without merging**